Core.Collectors/Config/ConfigManager.cs

```diff
@@ -156,6 +161,32 @@ protected JToken GetAuthenticationToken(string collectorType)
             return authenticationToken;
         }
 
+        public JToken GetApiDomainToken()
+        {
+            ValidateSettingsExist();
+
+            if (!this.apiDomainTokenMap.TryGetValue(DefaultKey, out JToken apiDomainToken))
```

Comment:
If we are only using DefaultKey (correct me if I am wrong) in this dictionary, then is there a reason to keep this as a dictionary? Can we e.g., use a simple string (or JToken) for the ApiDomainToken's value?

Reply:
This is something I went back and forth with in my head. I'm updating it to just a JToken in my newest commit.